content/support.rst

```diff
@@ -0,0 +1,193 @@
+	product.
+	e.g. Firewalls, Proxy, LDAP, Active Directory, Single Sign On, Management software etc.
+
+	Before releasing a new version, we run an automated test suite to validate new features
+	and check for regressions.
+	
+	By knowing details about your installations, we can configure the test suite to run
+	against a similar setup.
+
+	This information will not be shared with any other third party and will be used
+	solely for improving further support for you and development of the product.
+
+
+Using the installation details
+------------------------------
+
+Once the installation details are provided, recreate the issue in a similar
+environment.
+
+This should be created via a dedicated VM for testing / support activities.
+
+
+Chat
+====
+
+`Olark <https://www.olark.com>`_ is currently used for chat with our customers
+or possible customers.
+
+Each chat transcript is automatically forwarded to the Support email list.
+
+A chat can be transferred and other actions can be taken with the chat via
+Olark `commands <https://www.olark.com/help/commands>`_.
+
+
+Licenses and support contract
+=============================
+
+See the `Support options page <https://www.sftpplus.com/support/options.html>`_
+which describes the options available.
+
+If a support case goes beyond the standard option (such as a new
+product feature), the Sales team will determine the quote.
+
+See the `Life cycle page <https://www.sftpplus.com/product/life-cycle.html>`_ 
+for details about the product life cycle.
+
+If a feature request is obtained, check that it is included in the
+`Product roadmap <https://www.sftpplus.com/product/roadmap.html>`_ 
+
+
+Support Case Management
+=======================
+
+
+Working on a support case
+-------------------------
+
+Support cases can be received either through the online contact form,
+direct email or phone call.
+
+Email and support forms are the preferred methods.
+When we have a phone call, we request the customer at the beginning to do a
+follow up over email and confirm in writing.
+
+Support can be requested by partners or customers.
+
+When a support case is started, obtain and collate as much information as
+possible.
+This will help in potential follow-up with the rest of the Support team.
+
+
+Troubleshooting a Support Case
+------------------------------
+
+The `troubleshooting theory from CompTIA <http://certmag.com/guide-troubleshooting-theory-comptia-perspective/>`_ is a good overview when
+troubleshooting a support case:
+
+1. Identify the problem
+2. Establish a theory of probable cause (question the obvious)
+3. Test the theory to determine cause
+4. Establish a plan
+5. Determine system status
+6. Make a record
+
+
+Initial Assessment
+------------------
+
+When a support case has started, it should receive a response that it has
+been received and a case open within the same or next business day.
+
+The case will be allocated to an Owner, who should do the initial assessment:
```

Comment:
Who does the allocation?

The guide should be very clear, so in the case of which something goes wrong, or we don't know what to do we can read it and know exactly what to do who should do it.

Reply:
I think this section should be deleted..info is generic